runtime/src/bank_forks.rs

```diff
@@ -644,6 +651,12 @@ impl ForkGraph for BankForks {
     }
 }
 
+impl Drop for BankForks {
+    fn drop(&mut self) {
+        error!("BankForks::drop(): successfully dropped");
```

Comment:
note to self: now that, ProgramCache's BankForks circular-referenc is gone. maybe i can wire `prepare_to_drop` from here to exercise the shutdown code-path (esp in tests!)

Reply:
this is done: 163015ab20a7eadcf58742eee48d7944bb0016e5